stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -161,7 +159,6 @@ spec:
         name: data
         labels:
           app: {{ template "rabbitmq.name" . }}
-          chart: {{ template "rabbitmq.chart" .  }}
```

Comment:
This can also stay, per slack chat

Reply:
volumeClaimTemplates is immutable: this can, unfortunately, not stay.